setup.py

```diff
@@ -1,8 +1,14 @@
+import re
```

Comment:
We have the convention of importing Python modules in a single line to save space.

I note you do the same on line 7... :-)